--- a/client/cmd/root.go
+++ b/client/cmd/root.go
@@ -95,11 +95,11 @@ func Execute() error {
 }
 
 func init() {
-	defaultConfigPathDir = "/etc/netbird/"
-	defaultLogFileDir = "/var/log/netbird/"
+	defaultConfigPathDir = "/opt/etc/netbird/"
+	defaultLogFileDir = "/opt/var/log/netbird/"
 
-	oldDefaultConfigPathDir = "/etc/wiretrustee/"
-	oldDefaultLogFileDir = "/var/log/wiretrustee/"
+	oldDefaultConfigPathDir = "/opt/etc/wiretrustee/"
+	oldDefaultLogFileDir = "/opt/var/log/wiretrustee/"
 
 	switch runtime.GOOS {
 	case "windows":
@@ -118,7 +118,7 @@ func init() {
 	oldDefaultConfigPath = oldDefaultConfigPathDir + "config.json"
 	oldDefaultLogFile = oldDefaultLogFileDir + "client.log"
 
-	defaultDaemonAddr := "unix:///var/run/netbird.sock"
+	defaultDaemonAddr := "unix:///opt/var/run/netbird.sock"
 	if runtime.GOOS == "windows" {
 		defaultDaemonAddr = "tcp://127.0.0.1:41731"
 	}
--- a/client/configs/configs.go
+++ b/client/configs/configs.go
@@ -17,7 +17,7 @@ func init() {
 	case "windows":
 		StateDir = filepath.Join(os.Getenv("PROGRAMDATA"), "Netbird")
 	case "darwin", "linux":
-		StateDir = "/var/lib/netbird"
+		StateDir = "/opt/var/lib/netbird"
 	case "freebsd", "openbsd", "netbsd", "dragonfly":
 		StateDir = "/var/db/netbird"
 	}
--- a/client/iface/configurer/usp.go
+++ b/client/iface/configurer/usp.go
@@ -210,7 +210,7 @@ func (t *WGUSPConfigurer) Close() {
 	}
 
 	if runtime.GOOS == "linux" {
-		sockPath := "/var/run/wireguard/" + t.deviceName + ".sock"
+		sockPath := "/opt/var/run/wireguard/" + t.deviceName + ".sock"
 		if _, statErr := os.Stat(sockPath); statErr == nil {
 			_ = os.Remove(sockPath)
 		}
--- a/client/internal/routemanager/systemops/systemops_linux.go
+++ b/client/internal/routemanager/systemops/systemops_linux.go
@@ -29,7 +29,7 @@ const (
 	NetbirdVPNTableName = "netbird"
 
 	// rtTablesPath is the path to the file containing the routing table names.
-	rtTablesPath = "/etc/iproute2/rt_tables"
+	rtTablesPath = "/opt/etc/iproute2/rt_tables"
 
 	// ipv4ForwardingPath is the path to the file containing the IP forwarding setting.
 	ipv4ForwardingPath = "net.ipv4.ip_forward"
--- a/client/system/osrelease_unix.go
+++ b/client/system/osrelease_unix.go
@@ -11,9 +11,9 @@ import (
 )
 
 func readOsReleaseFile() (osName string, osVer string) {
-	file, err := os.Open("/etc/os-release")
+	file, err := os.Open("/opt/etc/os-release")
 	if err != nil {
-		log.Warnf("failed to open file /etc/os-release: %s", err)
+		log.Warnf("failed to open file /opt/etc/os-release: %s", err)
 		return "", ""
 	}
 	defer file.Close()
--- a/client/ui/client_ui.go
+++ b/client/ui/client_ui.go
@@ -103,7 +103,7 @@ func main() {
 
 // parseFlags reads and returns all needed command-line flags.
 func parseFlags() (daemonAddr string, showSettings, showNetworks, showDebug bool, errorMsg string, saveLogsInFile bool) {
-	defaultDaemonAddr := "unix:///var/run/netbird.sock"
+	defaultDaemonAddr := "unix:///opt/var/run/netbird.sock"
 	if runtime.GOOS == "windows" {
 		defaultDaemonAddr = "tcp://127.0.0.1:41731"
 	}
--- a/management/cmd/defaults.go
+++ b/management/cmd/defaults.go
@@ -1,13 +1,13 @@
 package cmd
 
 const (
-	defaultMgmtDataDir   = "/var/lib/netbird/"
-	defaultMgmtConfigDir = "/etc/netbird"
-	defaultLogDir        = "/var/log/netbird"
+	defaultMgmtDataDir   = "/opt/var/lib/netbird/"
+	defaultMgmtConfigDir = "/opt/etc/netbird"
+	defaultLogDir        = "/opt/var/log/netbird"
 
-	oldDefaultMgmtDataDir   = "/var/lib/wiretrustee/"
-	oldDefaultMgmtConfigDir = "/etc/wiretrustee"
-	oldDefaultLogDir        = "/var/log/wiretrustee"
+	oldDefaultMgmtDataDir   = "/opt/var/lib/wiretrustee/"
+	oldDefaultMgmtConfigDir = "/opt/etc/wiretrustee"
+	oldDefaultLogDir        = "/opt/var/log/wiretrustee"
 
 	defaultMgmtConfig    = defaultMgmtConfigDir + "/management.json"
 	defaultLogFile       = defaultLogDir + "/management.log"
--- a/signal/cmd/root.go
+++ b/signal/cmd/root.go
@@ -39,8 +39,8 @@ func Execute() error {
 
 func init() {
 	stopCh = make(chan int)
-	defaultLogFile = "/var/log/netbird/signal.log"
-	defaultSignalSSLDir = "/var/lib/netbird/"
+	defaultLogFile = "/opt/var/log/netbird/signal.log"
+	defaultSignalSSLDir = "/opt/var/lib/netbird/"
 
 	if runtime.GOOS == "windows" {
 		defaultLogFile = os.Getenv("PROGRAMDATA") + "\\Netbird\\" + "signal.log"
--- a/upload-server/server/local.go
+++ b/upload-server/server/local.go
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	defaultDir = "/var/lib/netbird"
+	defaultDir = "/opt/var/lib/netbird"
 	putHandler = "/{dir}/{file}"
 )
 
