--- a/cmd/root.go
+++ b/cmd/root.go
@@ -134,9 +134,8 @@ as FB_DISABLE_PREVIEW_RESIZE.
 If "--config" is not specified, File Browser will look for a configuration
 file named .filebrowser.{json, toml, yaml, yml} in the following directories:
 
-- ./
+- /opt/etc/filebrowser/
 - $HOME/
-- /etc/filebrowser/
 
 **Note:** Only the options listed below can be set via the config file or
 environment variables. Other configuration options live exclusively in the
--- a/cmd/utils.go
+++ b/cmd/utils.go
@@ -97,10 +97,8 @@ func initViper(cmd *cobra.Command) (*vip
 		if err != nil {
 			return nil, err
 		}
-		v.AddConfigPath(".")
+		v.AddConfigPath("/opt/etc/filebrowser/")
 		v.AddConfigPath(home)
-		v.AddConfigPath("/etc/filebrowser/")
-		v.SetConfigName(".filebrowser")
 	} else {
 		v.SetConfigFile(cfgFile)
 	}
