--- a/cmd/root.go
+++ b/cmd/root.go
@@ -91,9 +91,9 @@ environment variables or configuration f
 If you don't set "config", it will look for a configuration file called
 .filebrowser.{json, toml, yaml, yml} in the following directories:
 
-- ./
+- /opt/etc/filebrowser/
 - $HOME/
-- /etc/filebrowser/
+- ./
 
 The precedence of the configuration values are as follows:
 
@@ -403,9 +403,9 @@ func initConfig() {
 	if cfgFile == "" {
 		home, err := homedir.Dir()
 		checkErr(err)
-		v.AddConfigPath(".")
+		v.AddConfigPath("/opt/etc/filebrowser/")
 		v.AddConfigPath(home)
-		v.AddConfigPath("/etc/filebrowser/")
+		v.AddConfigPath(".")
 		v.SetConfigName(".filebrowser")
 	} else {
 		v.SetConfigFile(cfgFile)
