--- a/conf/defaults.ini
+++ b/conf/defaults.ini
@@ -16,19 +16,19 @@ force_migration = false
 #################################### Paths ###############################
 [paths]
 # Path to where grafana can store temp files, sessions, and the sqlite3 db (if that is used)
-data = data
+data = /opt/var/lib/grafana
 
 # Temporary files in `data` directory older than given duration will be removed
 temp_data_lifetime = 24h
 
 # Directory where grafana can store logs
-logs = data/log
+logs = /opt/var/log/grafana
 
 # Directory where grafana will automatically scan and look for plugins
-plugins = data/plugins
+plugins = /opt/var/lib/grafana/plugins
 
 # folder that contains provisioning config files that grafana will apply on startup and while running.
-provisioning = conf/provisioning
+provisioning = /opt/etc/grafana/provisioning
 
 #################################### Server ##############################
 [server]
@@ -80,7 +80,7 @@ socket_gid = -1
 socket_mode = 0660
 
 # Unix socket path
-socket = /tmp/grafana.sock
+socket = /opt/var/run/grafana.sock
 
 # CDN Url
 cdn_url =
@@ -861,7 +861,7 @@ skip_org_role_sync = false
 #################################### Auth LDAP ###########################
 [auth.ldap]
 enabled = false
-config_file = /etc/grafana/ldap.toml
+config_file = /opt/etc/grafana/ldap.toml
 allow_sign_up = true
 skip_org_role_sync = false
 
