--- a/src/github.com/tailscale/tailscale/logpolicy/logpolicy.go
+++ b/src/github.com/tailscale/tailscale/logpolicy/logpolicy.go
@@ -387,7 +387,7 @@ func tryFixLogStateLocation(dir, cmdname
 	existsInCache := false
 	cacheDir := os.Getenv("CACHE_DIRECTORY")
 	if cacheDir != "" {
-		existsInCache, err = checkExists("/var/cache/tailscale")
+		existsInCache, err = checkExists("/opt/var/cache/tailscale")
 		if err != nil {
 			log.Printf("checking for configs in %s: %v", cacheDir, err)
 		}
--- a/src/github.com/tailscale/tailscale/paths/paths.go
+++ b/src/github.com/tailscale/tailscale/paths/paths.go
@@ -42,8 +42,8 @@ func DefaultTailscaledSocket() string {
 	case distro.Gokrazy:
 		return "/perm/tailscaled/tailscaled.sock"
 	}
-	if fi, err := os.Stat("/var/run"); err == nil && fi.IsDir() {
-		return "/var/run/tailscale/tailscaled.sock"
+	if fi, err := os.Stat("/opt/var/run"); err == nil && fi.IsDir() {
+		return "/opt/var/run/tailscale/tailscaled.sock"
 	}
 	return "tailscaled.sock"
 }
--- a/src/github.com/tailscale/tailscale/paths/paths_unix.go
+++ b/src/github.com/tailscale/tailscale/paths/paths_unix.go
@@ -24,7 +24,7 @@ func init() {
 func statePath() string {
 	switch runtime.GOOS {
 	case "linux":
-		return "/var/lib/tailscale/tailscaled.state"
+		return "/opt/var/lib/tailscale/tailscaled.state"
 	case "freebsd", "openbsd":
 		return "/var/db/tailscale/tailscaled.state"
 	case "darwin":
