--- a/src/github.com/rclone/rclone/fs/config/config.go
+++ b/src/github.com/rclone/rclone/fs/config/config.go
@@ -105,10 +105,10 @@
 	// CacheDir points to the cache directory.  Users of this
 	// should make a subdirectory and use MkdirAll() to create it
 	// and any parents.
-	CacheDir = makeCacheDir()
+	CacheDir = "/opt/var/rclone"
 
 	// ConfigPath points to the config file
-	ConfigPath = makeConfigPath()
+	ConfigPath = "/opt/etc/rclone.config"
 
 	// Password can be used to configure the random password generator
 	Password = random.Password
