--- a/fs/config/config.go
+++ b/fs/config/config.go
@@ -125,8 +125,8 @@ func init() {
 	fs.ConfigFileHasSection = func(section string) bool {
 		return LoadedData().HasSection(section)
 	}
-	configPath = makeConfigPath()
-	cacheDir = makeCacheDir() // Has fallback to tempDir, so set that first
+	configPath = "/opt/etc/rclone.config"
+	cacheDir = "/opt/var/cache/rclone"
 	data = newDefaultStorage()
 }
 
