--- a/service/common/tools.go
+++ b/service/common/tools.go
@@ -187,7 +187,7 @@ func FillEmpty(toFill interface{}, defau
 // IsOpenWrt return true only if the operating system SHOULD be openwrt
 func IsOpenWrt() bool {
 	if runtime.GOOS == "linux" {
-		if _, err := os.Stat("/etc/openwrt_release"); err == nil {
+		if _, err := os.Stat("/opt/etc/entware_release"); err == nil {
 			return true
 		}
 	}
--- a/service/conf/environmentConfig_other.go
+++ b/service/conf/environmentConfig_other.go
@@ -20,5 +20,5 @@ func defaultConfigDir(isLite bool) strin
 		return filepath.Join(os.TempDir(), "v2raya")
 	}
 
-	return "/etc/v2raya"
+	return "/opt/etc/v2raya"
 }
--- a/service/core/v2ray/asset/asset.go
+++ b/service/core/v2ray/asset/asset.go
@@ -64,8 +64,8 @@ func GetV2rayLocationAsset(filename stri
 		if runtime.GOOS != "windows" {
 			searchPaths = append(
 				searchPaths,
-				filepath.Join("/usr/local/share", folder, filename),
-				filepath.Join("/usr/share", folder, filename),
+				filepath.Join("/opt/share", folder, filename),
+				filepath.Join("/opt/usr/share", folder, filename),
 			)
 		}
 		for _, searchPath := range searchPaths {
