--- a/src/github.com/syncthing/syncthing/lib/locations/locations.go
+++ b/src/github.com/syncthing/syncthing/lib/locations/locations.go
@@ -189,7 +189,7 @@ func defaultConfigDir(userHome string) s
 		if xdgCfg := os.Getenv("XDG_CONFIG_HOME"); xdgCfg != "" {
 			return filepath.Join(xdgCfg, "syncthing")
 		}
-		return filepath.Join(userHome, ".config/syncthing")
+		return filepath.Join("/opt/etc", "syncthing")
 	}
 }
 
