--- a/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/calmh/du/diskusage_posix.go
+++ b/src/github.com/syncthing/syncthing/Godeps/_workspace/src/github.com/calmh/du/diskusage_posix.go
@@ -5,6 +5,7 @@
 import (
 	"path/filepath"
 	"syscall"
+	"runtime"
 )
 
 // Get returns the Usage of a given path, or an error if usage data is
@@ -15,6 +16,14 @@
 	if err != nil {
 		return Usage{}, err
 	}
+
+	if (runtime.GOARCH=="mipso32") {
+		stat.Bavail  = 10000
+		stat.Bfree   = 10000
+		stat.Blocks  = 10000
+		stat.Bsize   = 4096
+	}
+
 	u := Usage{
 		FreeBytes:  int64(stat.Bfree) * int64(stat.Bsize),
 		TotalBytes: int64(stat.Blocks) * int64(stat.Bsize),
