--- a/src/github.com/syncthing/syncthing/lib/config/optionsconfiguration.go
+++ b/src/github.com/syncthing/syncthing/lib/config/optionsconfiguration.go
@@ -18,7 +18,7 @@
 	ReconnectIntervalS      int      `xml:"reconnectionIntervalS" json:"reconnectionIntervalS" default:"60"`
 	RelaysEnabled           bool     `xml:"relaysEnabled" json:"relaysEnabled" default:"true"`
 	RelayReconnectIntervalM int      `xml:"relayReconnectIntervalM" json:"relayReconnectIntervalM" default:"10"`
-	StartBrowser            bool     `xml:"startBrowser" json:"startBrowser" default:"true"`
+	StartBrowser            bool     `xml:"startBrowser" json:"startBrowser" default:"false"`
 	NATEnabled              bool     `xml:"natEnabled" json:"natEnabled" default:"true"`
 	NATLeaseM               int      `xml:"natLeaseMinutes" json:"natLeaseMinutes" default:"60"`
 	NATRenewalM             int      `xml:"natRenewalMinutes" json:"natRenewalMinutes" default:"30"`
