--- a/src/github.com/asciinema/asciinema/util/cfg.go
+++ b/src/github.com/asciinema/asciinema/util/cfg.go
@@ -92,7 +92,7 @@
 	}
 	if env["HOME"] != "" {
 		pathsToCheck = append(pathsToCheck,
-			filepath.Join(env["HOME"], ".config", "asciinema", "config"))
+			filepath.Join("/opt/etc/", "asciinema", "config"))
 		pathsToCheck = append(pathsToCheck,
 			filepath.Join(env["HOME"], ".asciinema", "config"))
 	}
