--- a/internal/mount/mount_posix_webdav_helper_linux.go
+++ b/internal/mount/mount_posix_webdav_helper_linux.go
@@ -6,18 +6,18 @@ import (
 )
 
 func mountWebDavHelper(ctx context.Context, url, path string) error {
-	mount := exec.Command("/usr/bin/mount", "-t", "davfs", "-r", url, path)
+	mount := exec.Command("/opt/bin/mount", "-t", "davfs", "-r", url, path)
 	if err := mount.Run(); err != nil {
-		log(ctx).Errorf("mount command failed: %v. Cowardly refusing to run with root permissions. Try \"sudo /usr/bin/mount -t davfs -r %s %s\"\n", err, url, path)
+		log(ctx).Errorf("mount command failed: %v. Cowardly refusing to run with root permissions. Try \"sudo /opt/bin/mount -t davfs -r %s %s\"\n", err, url, path)
 	}
 
 	return nil
 }
 
 func unmountWebDevHelper(ctx context.Context, path string) error {
-	unmount := exec.Command("/usr/bin/umount", path)
+	unmount := exec.Command("/opt/bin/umount", path)
 	if err := unmount.Run(); err != nil {
-		log(ctx).Errorf("umount command failed: %v. Cowardly refusing to run with root permissions. Try \"sudo /usr/bin/umount %s\"\n", err, path)
+		log(ctx).Errorf("umount command failed: %v. Cowardly refusing to run with root permissions. Try \"sudo /opt/bin/umount %s\"\n", err, path)
 	}
 
 	return nil
--- a/internal/ospath/ospath_xdg.go
+++ b/internal/ospath/ospath_xdg.go
@@ -12,12 +12,12 @@ func init() {
 	if os.Getenv("XDG_CONFIG_HOME") != "" {
 		userSettingsDir = os.Getenv("XDG_CONFIG_HOME")
 	} else {
-		userSettingsDir = filepath.Join(os.Getenv("HOME"), ".config")
+		userSettingsDir = filepath.Join("/opt", "/etc")
 	}
 
 	if os.Getenv("XDG_CACHE_HOME") != "" {
 		userLogsDir = os.Getenv("XDG_CACHE_HOME")
 	} else {
-		userLogsDir = filepath.Join(os.Getenv("HOME"), ".cache")
+		userLogsDir = filepath.Join("/opt/var", "/log")
 	}
 }
