--- a/src/github.com/ncw/rclone/fs/config/config.go
+++ b/src/github.com/ncw/rclone/fs/config/config.go
@@ -63,12 +63,12 @@
 	configData *goconfig.ConfigFile
 
 	// ConfigPath points to the config file
-	ConfigPath = makeConfigPath()
+	ConfigPath = "/opt/etc/rclone.config"
 
 	// CacheDir points to the cache directory.  Users of this
 	// should make a subdirectory and use MkdirAll() to create it
 	// and any parents.
-	CacheDir = makeCacheDir()
+	CacheDir = "/opt/var/rclone"
 
 	// Config is the global config
 	Config = &ConfigInfo{}
